### ISSUE TITLE
SCG taxonomy `--update-profile-db-with-taxonomy` fails in anvi'o v6.2

### ISSUE THREAD

**Issue by Zhanglin1990** [merenlab/anvio#1412]
Dear Anvio team,

I installed the anvio-6.2 on Linux through conda. This is the information about my anvio version:

anvi-self-test --version
Anvi'o version ...............................: esther (v6.2)
Profile DB version ...........................: 31
Contigs DB version ...........................: 14
Pan DB version ...............................: 13
Genome data storage version ..................: 6
Auxiliary data storage version ...............: 2
Structure DB version .........................: 1

When I run the command: **anvi-estimate-scg-taxonomy -c 02_CONTIG/contigs.db -p 05_PROFILE/PROFILE.db --metagenome-mode --compute-scg-coverages --update-profile-db-with-taxonomy**,  I faced this issue:
✖ anvi-estimate-scg-taxonomy encountered an error after 0:00:16.061642
Traceback (most recent call last):
  File "/home/lupin/miniconda3/envs/anvio-6.2/bin/anvi-estimate-scg-taxonomy", line 103, in <module>
    main(args)
  File "/home/lupin/miniconda3/envs/anvio-6.2/lib/python3.6/site-packages/anvio/terminal.py", line 748, in wrapper
    program_method(*args, **kwargs)
  File "/home/lupin/miniconda3/envs/anvio-6.2/bin/anvi-estimate-scg-taxonomy", line 40, in main
    t.estimate()
  File "/home/lupin/miniconda3/envs/anvio-6.2/lib/python3.6/site-packages/anvio/scgtaxonomyops.py", line 1459, in estimate
    self.add_taxonomy_as_additional_layer_data(scg_taxonomy_super_dict)
  File "/home/lupin/miniconda3/envs/anvio-6.2/lib/python3.6/site-packages/anvio/scgtaxonomyops.py", line 1629, in add_taxonomy_as_additional_layer_data
    for level in constants.ctx.levels_of_taxonomy[::-1]:
AttributeError: module 'anvio.constants' has no attribute 'ctx'

I didn't find a similar problem, Could you please give me some advices?  

Thank you very much.

**Comment by meren** [comment]
Hi @Zhanglin1990,

This is certainly a bug in `v6.2` :/ It is now fixed in `master` and will be corrected in `v6.3`. Thank you very much for your report.

If you want to get rid of this error you can open this file in a text editor:

```
/home/lupin/miniconda3/envs/anvio-6.2/lib/python3.6/site-packages/anvio/scgtaxonomyops.py
```

And then search for this term:

```
constants.ctx.levels_of_taxonomy
```

just to replace it with this (please note that there are multiple occurrences of it):

```
self.ctx.levels_of_taxonomy
```

Apologies for this mistake.


Best wishes,

**Comment by Zhanglin1990** [comment]
Thank you very much for your kindly help. This problem has been solved. Thanks again.
